samples/download_view_image.py

```diff
@@ -0,0 +1,68 @@
+####
+# This script demonstrates how to use the Tableau Server Client
+# to download a high resolution image of a view from Tableau Server.
+#
+# For more information, refer to the documentations on 'Query View Image'
+# (https://onlinehelp.tableau.com/current/api/rest_api/en-us/help.htm)
+#
+# To run the script, you must have installed Python 2.7.X or 3.3 and later.
+####
+
+import argparse
+import getpass
+import logging
+
+import tableauserverclient as TSC
+
+
+def main():
+
+    parser = argparse.ArgumentParser(description='Query View Image From Server')
+    parser.add_argument('--server', '-s', required=True, help='server address')
+    parser.add_argument('--site-id', '-si', required=False,
+                        help='content url for site the view is on')
+    parser.add_argument('--username', '-u', required=True, help='username to sign into server')
+    parser.add_argument('--view-name', '-v', required=True,
+                        help='name of view to download an image of')
+    parser.add_argument('--filepath', '-f', required=True, help='filepath to save the image returned')
+    parser.add_argument('--logging-level', '-l', choices=['debug', 'info', 'error'], default='error',
+                        help='desired logging level (set to error by default)')
+
+    args = parser.parse_args()
+
+    password = getpass.getpass("Password: ")
+
+    # Set logging level based on user input, or error by default
+    logging_level = getattr(logging, args.logging_level.upper())
+    logging.basicConfig(level=logging_level)
+
+    # Step 1: Sign in to server.
+    site_id = args.site_id
+    if not site_id:
+        site_id = ""
+    tableau_auth = TSC.TableauAuth(args.username, password, site_id=site_id)
+    server = TSC.Server(args.server)
+    # The new endpoint was introduced in Version 2.5
+    server.version = 2.5
+
+    with server.auth.sign_in(tableau_auth):
+        # Step 2: Query for the view that we want an image of
+        req_option = TSC.RequestOptions()
+        req_option.filter.add(TSC.Filter(TSC.RequestOptions.Field.Name,
+                                         TSC.RequestOptions.Operator.Equals, args.view_name))
+        all_views, pagination_item = server.views.get(req_option)
+        if not all_views:
+            raise LookupError("View with the specified name was not found.")
+        view_item = all_views[0]
+
+        # Step 3: Query the image endpoint and save the image to the specified location
+        image_req_option = TSC.ImageRequestOptions(imageresolution=TSC.ImageRequestOptions.Resolution.High)
```

Comment:
Thanks @LGraber , I opened #114 to track this idea -- I've been googling how some other libraries do it for inspiration (nothing struck me yet), but that'll make it so we don't lose the convo in this PR